napari/components/add_layers_mixin.py

```diff
@@ -1,8 +1,9 @@
+import inspect
 import itertools
 from functools import lru_cache
 from logging import getLogger
 from os import fspath
-from typing import Any, Dict, List, Optional, Sequence, Union, Set
+from typing import Any, Dict, List, Optional, Sequence, Set, Union
```

Comment:
Hahaha this change definitely reveals a personality type. :joy:

Reply:
sorry :/ motor memory